src/types/script.zig

```diff
@@ -0,0 +1,56 @@
+const std = @import("std");
+const CompactSizeUint = @import("bitcoin-primitives").types.CompatSizeUint;
+const readBytesExact = @import("../util/mem/read.zig").readBytesExact;
+
+bytes: []u8,
+allocator: std.mem.Allocator,
+
+const Self = @This();
+
+/// Initialize a new script
+pub fn init(allocator: std.mem.Allocator) !Self {
+    return Self{
+        .bytes = try allocator.alloc(u8, 0),
+        .allocator = allocator,
+    };
+}
+
+/// Deinitialize the script
+pub fn deinit(self: *Self) void {
+    self.allocator.free(self.bytes);
+}
+
+/// Add data to the script
+pub fn push(self: *Self, data: []const u8) !void {
```

Comment:
but are we really using this method?

Reply:
we are using it in the tests, but I'm not the one who made that code, so I didn't check if this function was used anywhere

Reply:
We are using this in AddOutput (Transaction), as well as in a few tests